lib/src/commands/packages.dart

```diff
@@ -65,7 +65,7 @@ class PackagesGetCommand extends Command<int> {
         await Flutter.packagesGet(
           cwd: targetPath,
           recursive: recursive,
-          progress: _logger.progress,
+          progress: (value) => _logger.progress(value),
```

Comment:
How does this complete?

Reply:
oops good catch! just updated to proxy to `complete`

Reply:
I think in the long run we might want to provide the whole progress object? As the tests can fail and we then can show that better? For now, LGTM!

Reply:
yeah I'm going to close this for now in favor of https://github.com/VeryGoodOpenSource/very_good_cli/pull/411 and we can properly upgrade this week